go/Makefile

```diff
@@ -1,5 +1,9 @@
 install-tools:
-	@cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+	go install github.com/vakenbolt/go-test-report@v0.9.3
```

Comment:
is there a way in go to do like in python `pip install -r requirements.txt`?

Reply:
Not as far as I know. The tools listed here are CI tools. One recommended way to specify tool dependencies is to have the CI tools specified in go.mod. However, this was not working for me with CI because go 1.18 and go 1.21 require different versions of CI tools. Old tools did not work on go 1.21 and new tools did not work on go 1.18. I've looked around and have not been able to find recommendations on how to handle this scenario. The solution I've landed on so far is to have different make targets for installing tools on go 1.18 vs go 1.20